openedx_caliper_tracking/tests/test_caliper_kafka.py

```diff
@@ -5,11 +5,13 @@
 import json
 
 import mock
-from django.conf import settings
 from django.test import TestCase, override_settings
 from kafka.errors import KafkaError
+
 from openedx_caliper_tracking.processor import CaliperProcessor
-from openedx_caliper_tracking.tasks import _host_not_found
+from openedx_caliper_tracking.tasks import (_host_not_found, deliver_caliper_event_to_kafka,
```

Comment:
```suggestion
from openedx_caliper_tracking.tasks import (host_not_found, deliver_caliper_event_to_kafka,
```
as this function is not private please rm the `_`